packages/features/ee/teams/components/MemberListItem.tsx

```diff
@@ -142,242 +262,420 @@ export default function MemberListItem(props: Props) {
-  const bookerUrlWithoutProtocol = bookerUrl.replace(/^https?:\/\//, "");
-  const bookingLink = !!props.member.username && `${bookerUrlWithoutProtocol}/${props.member.username}`;
-  const isAdmin = props.team && ["ADMIN", "OWNER"].includes(props.team.membership?.role);
-  const appList = props.connectedApps?.map(({ logo, name, externalId }) => {
-    return logo ? (
-      externalId ? (
-        <div className="ltr:mr-2 rtl:ml-2 ">
-          <Tooltip content={externalId}>
-            <img className="h-5 w-5" src={logo} alt={`${name} logo`} />
-          </Tooltip>
-        </div>
-      ) : (
-        <div className="ltr:mr-2 rtl:ml-2">
-          <img className="h-5 w-5" src={logo} alt={`${name} logo`} />
-        </div>
-      )
-    ) : null;
-  });
+  const totalDBRowCount = data?.pages?.[0]?.meta?.totalRowCount ?? 0;
 
-  return (
-    <li className="divide-subtle divide-y px-5">
-      <div className="my-4 flex justify-between">
-        <div className="flex w-full flex-col justify-between truncate sm:flex-row">
-          <div className="flex">
-            <UserAvatar
-              noOrganizationIndicator
-              size="sm"
-              user={props.member}
-              className="h-10 w-10 rounded-full"
-            />
-            <div className="ms-3 inline-block">
-              <div className="mb-1 flex" data-testid={`member-${props.member.username}`}>
-                <span data-testid="member-name" className="text-default mr-2 text-sm font-bold leading-4">
-                  {name}
-                </span>
-                {!props.member.accepted && (
-                  <TeamPill data-testid="member-pending" color="orange" text={t("pending")} />
-                )}
-                {isAdmin && props.member.accepted && appList}
-                {props.member.role && <TeamRole data-testid="member-role" role={props.member.role} />}
-              </div>
-              <div className="text-default flex items-center">
-                <span
-                  className=" block text-sm"
-                  data-testid={
-                    props.member.accepted
-                      ? "member-email"
-                      : `email-${props.member.email.replace("@", "")}-pending`
-                  }
-                  data-email={props.member.email}>
-                  {props.member.email}
-                </span>
-                {bookingLink && (
-                  <>
-                    <span className="text-default mx-2 block">•</span>
-                    <a
-                      target="_blank"
-                      href={`${bookerUrl}/${props.member.username}`}
-                      className="text-default block text-sm">
-                      {bookingLink}
-                    </a>
-                  </>
-                )}
+  const memorisedColumns = useMemo(() => {
+    const cols: ColumnDef<User>[] = [
+      // Disabling select for this PR: Will work on actions etc in a follow up
```

Comment:
As far as I can see this has been implemented right? Let's remove this comment then
```suggestion
```